src/manage_nsfs/versions.js

```diff
@@ -0,0 +1,82 @@
+/* Copyright (C) 2024 NooBaa */
+'use strict';
+
+const dbg = require('../util/debug_module')(__filename);
+const config = require('../../config');
+const pkg = require('../../package.json');
+const http_utils = require('../util/http_utils');
+const buffer_utils = require('../util/buffer_utils');
+const { write_stdout_response } = require('./manage_nsfs_cli_utils');
+const { ManageCLIResponse } = require('./manage_nsfs_cli_responses');
+
+////////////////////////
+// VERSIONS MANAGEMENT //
+////////////////////////
+
+/**
+ * versions_management 
+ */
+async function versions_management(config_fs) {
+    const system_json = await config_fs.get_system_config_file({ silent_if_missing: true });
+    const versions = {
+        rpm_source_code_versions: {
+            package_version: pkg.version,
+            config_fs_version: config_fs.config_dir_version,
+        },
+        host_running_service_versions: await get_running_service_versions(),
+        config_dir_version: system_json?.config_directory.config_dir_version || 'unknown'
+    };
+
+    const response = { code: ManageCLIResponse.VersionsStatus, detail: versions };
```

Comment:
@romayalon 
suggestion: will it help if the rpm and host versions are different and the version status returns an error message mentioning that conflict?